docs/administration-guide/06-collections.md

```diff
@@ -49,13 +53,38 @@ Each user has a personal collection where they're always allowed to save things,
 A personal collection works just like any other collection except that its permissions can't be changed. If a sub-collection within a personal collection is moved to a different collection, it will inherit the permissions of that collection.
 
 ### Archiving collections
+
 Users with curate permission for a collection can archive collections. Click the edit icon in the top-right of the collection screen and select `Archive this collection` to archive it. This will also archive all questions, dashboards, pulses, and all other sub-collections and their contents. Importantly, this will also remove any archived questions from all dashboards and Pulses that use them.
 
 **Note:** the "Our analytics" collection and personal collections can't be archived.
 
-You can always *unarchive* things by clicking on the More menu from a collection and selecting `View the archive`, then clicking the un-archive button next to an archived item. Questions within archived collections are not individually listed in the archive, so if you want to unarchive a specific question from an archived collection, you have to unarchive that whole collection.
+You can always _unarchive_ things by clicking on the More menu from a collection and selecting `View the archive`, then clicking the un-archive button next to an archived item. Questions within archived collections are not individually listed in the archive, so if you want to unarchive a specific question from an archived collection, you have to unarchive that whole collection.
```

Comment:
`You can always _unarchive_ things by clicking on the More menu from a collection and selecting View the archive`

That's not how you get to the archive anymore: there's now a link in the bottom-left of the collections list sidebar.